transport-native-io_uring/src/test/java/io/netty/channel/uring/IoUringDomainSocketFdTest.java

```diff
@@ -145,13 +151,17 @@ public void channelReadComplete(ChannelHandlerContext ctx) throws Exception {
 
         FileDescriptor fd = recvFdFuture.get();
         assertTrue(fd.isOpen());
+        assertNotEquals(0, fd.intValue());
         fd.close();
         assertFalse(fd.isOpen());
 
-        ByteBuf recvBuffer = recvByteBufFuture.get();
-        assertEquals(expected, recvBuffer.toString(Charset.defaultCharset()));
-
-        cc.close().sync();
-        sc.close().sync();
+       try {
+           ByteBuf recvBuffer = recvByteBufFuture.get();
+           assertEquals(expected, recvBuffer.toString(Charset.defaultCharset()));
+           cc.close().sync();
+           sc.close().sync();
+       } finally {
+           ReferenceCountUtil.release(fd);
```

Comment:
`FileDescriptor` is not reference counted. It's also closed on line 155. You should be releasing the `recvBuffer` here instead.